python/cugraph/proto/structure/bicliques.py

```diff
@@ -0,0 +1,326 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# Import needed libraries
+import cudf
+import numpy as np
+
+
+
+def find_bicliques(df, k, 
+              offset=0, 
+              max_iter=-1, 
+              support=1.0, 
+              min_features=1, 
+              min_machines=10) :
+    """
+    Find the top k maximal bicliques
+ 
+    Parameters
+    ---------- 
+    df :  cudf:DataFrame
+        A dataframe containing the bipartite graph edge list
+        Columns must be called 'src', 'dst', and 'flag'
+        
+    k  :  int
+        The max number of bicliques to return
+        -1 mean all
+    
+    offset : int
+      
+    
+    
+    
+    
+    Returns
+    -------     
+    B : cudf.DataFrame      
```

Comment:
There is some application-specific naming (eg. machines /  features). It is probably fine for the proto but we may want to make it more generic later on.